src/main/java/org/symphonyoss/symphony/messageml/elements/Element.java

```diff
@@ -302,6 +329,27 @@ public String asText() {
     return b.toString();
   }
 
+  /**
+   * This method checks recursively inside all children of an element and returns the count of a give type passed
```

Comment:
recursively caught my attention :), should we say it is a breadth-first traversal of a tree of elements instead?

Reply:
yes, done in https://github.com/symphonyoss/messageml-utils/pull/266/commits/bedab46386b8a9405936838f80189140567d89d4